CCPAConsentViewController/Classes/MessageWebViewController.swift

```diff
@@ -46,14 +46,16 @@ class MessageWebViewController: MessageViewController, WKUIDelegate, WKNavigatio
     let propertyId: Int
     let pmId: String
     let consentUUID: ConsentUUID?
+    let userConsent: UserConsent?
```

Comment:
I think it'd be better if `UserConsent` wasn't optional. This way, the `ConsentViewController` will be responsible for ensuring there's userconsent (even if it's a `UserConsent.rejectedNone()`) before instantiating the `MessageViewController`. What you think?

Reply:
@andresilveirah you are correct as we have `UserConsent.rejectedNone()`, we can UserConsent without optional. I  am going to remove that optional thing